Roles/Neutral/Shroud.cs

```diff
@@ -74,7 +74,11 @@ public static bool OnCheckMurder(PlayerControl killer, PlayerControl target)
         killer.SetKillCooldown();
 
         Utils.NotifyRoles(SpecifySeer: killer, SpecifyTarget: target, ForceLoop: true);
-
+        if (target.Is(CustomRoles.NiceMini) && Mini.Age < 18)
+        {
+            killer.Notify(Utils.ColorString(Utils.GetRoleColor(CustomRoles.NiceMini), GetString("CantShroud")));
+            return;
```

Comment:
you can't return nothing on a boolean function, also this dosen't do anything because it's called after the fact target has been made shrouded.